pdns/pdns_recursor.cc

```diff
@@ -2571,8 +2578,24 @@ static void houseKeeping(void *)
           g_log<<Logger::Error<<"Exception while performing security poll"<<endl;
         }
       }
+
+      if (!luaconfsLocal->trustAnchorFileInfo.fname.empty() && luaconfsLocal->trustAnchorFileInfo.interval != 0 &&
+          g_now.tv_sec - last_trustAnchorUpdate >= (luaconfsLocal->trustAnchorFileInfo.interval * 3600)) {
+        g_log<<Logger::Debug<<"Refreshing Trust Anchors from file"<<endl;
+        try {
+          map<DNSName, dsmap_t> dsAnchors;
```

Comment:
We might want to document that, when this feature is enabled, any `TA` added manually via `rec_control add-ta` will get overridden after a while.

Reply:
done!